### ISSUE TITLE
Request: Modify unix permissions on a file or directory

### ISSUE THREAD

**Issue by nsc-jens** [dCache/dcache-view#203]
We are trying to set up dcache-view as a general file access method for Swestore. There are a few things in the web GUI that we miss.

One very requested feature is a simple way to change the permissions on a file or directory. If this is done on a directory, it would be nice if it would be recursive. Or if you could selec and change multiple files at the same time. But on the whole, this is a tricky thing to get right.

**Comment by paulmillar** [comment]
Hi Jens,

Do you have an idea in which way the user would describe the changes to the file's or directory's permissions?

For example, the user could enter an octal number (e.g., `0644`, `0400`), a modifier description (e.g., `a+r`), or some GUI-based selection.  The GUI approach is perhaps the easiest for the average user, but also the most time-consuming.

**Comment by nsc-jens** [comment]
No, I realize this is a super complex thing to implement (on the GUI side). For a single file this is rather trivial. Right click, select "Change permissions" and select the right check boxes in an array. It quickly becomes very complicated if you want to change ten files with different initial permissions and/or a director structure. I guess the best start is to shop around and see if any other file browser has made a decent attempt on this.

I'm happy with the single file/dir change possibility which would solve the case with "I want to open my storage for anonymous read-only access".
